meson_options.txt

```diff
@@ -2,3 +2,15 @@ option('pg_config',
        type: 'string',
        value: '',
        description: 'Path to pg_config')
+option('msgpack',
+       type: 'feature',
+       value: 'disabled',
+       description: 'Enable MessagePack support')
+option('msgpack_package_name',
+       type: 'string',
+       value: 'msgpack',
+       description: 'MessagePack package name (e.g. "msgpack-c" on macOS)')
```

Comment:
We should detect it automatically like Groonga does.

Reply:
fix: [62dbf95](https://github.com/pgroonga/pgroonga/pull/750/commits/62dbf9598951c96f502fec7fc30176275fb70b0a) Sure.